multiapps-controller-core/src/test/resources/mta/shine/services.json

```diff
@@ -33,6 +35,8 @@
     "type" : "user_provided_service_instance",
     "alternativeLabels": [],
     "resourceName": "uaa",
-    "shouldIgnoreUpdateErrors": false
+	"shouldSkipCredentialsUpdate": false,
```

Comment:
There are some files with messed up indentation. Please fix it.

Reply:
Done